### PR TITLE
fix: prevent notice

### PR DESCRIPTION
**Pull request by vladutilie**
Access to the `post_type` property from a non-object is prevented.

Fix for: #37.

### CODE REVIEW

File: classes/AddonsManager.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public static function register() {
 		/**
 		 * If single course and built with elementor then remove description addon for avoiding the_content overlap
 		 */
-		if ( $post->post_type && $post->post_type == tutor()->course_post_type ) {
+		if ( isset( $post->post_type ) && $post->post_type == tutor()->course_post_type ) {
```
Comment (harunollyo) [inline classes/AddonsManager.php]: Use strict compare `===`

**Review by harunollyo — CHANGES_REQUESTED** [review]

File: classes/AddonsManager.php (hunk 1 above)
Comment (vladutilie) [reply classes/AddonsManager.php]: Done, but that was not the point of the problem. :)

**Comment by harunollyo** [comment]
Issue resolved in dev brunch

**Comment by vladutilie** [comment]
Oh, my bad for the PR in the master 🤦‍♂️

**Pull request closed without merging**